lightning/src/chain/channelmonitor.rs

```diff
@@ -1855,6 +1855,16 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitor<Signer> {
 		spendable_outputs
 	}
 
+	pub(crate) fn is_stale(&self) -> bool {
```

Comment:
@TheBlueMatt I would appreciate a bit of guidance here.

Is this function in the right place? and if so, how do I access the channel data here?